src/ORM/DataObject.php

```diff
@@ -1238,6 +1238,15 @@ public function forceChange()
     public function validate()
     {
         $result = ValidationResult::create();
+        // Call DBField::validate() on every DBField
+        $specs = static::getSchema()->fieldSpecs(static::class);
```

Comment:
```suggestion
        $specs = DataObject::getSchema()->fieldSpecs(static::class);
```
We can technically call `static::getSchema()` and that method can _technically_ be overridden.... but we tend to pretend that method is only available on `DataObject` directly. I had to mentally translate `static::getSchema()` into `DataObject::getSchema()` because I don't think I've _ever_ seen it called that way.

That said, this is very much a subjective suggestion so feel free to ignore it.

Reply:
Removed